pytorch_lightning/profiler/profilers.py

```diff
@@ -416,8 +399,10 @@ def __init__(
             self.on_train_start(local_rank=local_rank)
             self.on_train_start = super().on_train_start
 
-    def on_train_start(self, local_rank: Optional[str] = None):
+    def on_train_start(self, local_rank: Optional[str] = None, dir_path: str = None):
```

Comment:
what is this `dir_path` for? mind add a description...

Reply:
Done !